add_to_app/MultipleFluttersIos/MultipleFluttersIos/SingleFlutterViewController.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+import Flutter
+
+class SingleFlutterViewController : FlutterViewController, DataModelObserver {
+  var _channel : FlutterMethodChannel?
```

Comment:
this is one area kotlin does so much better. Though here this can still be 
`private lazy var channel: FlutterMethodChannel = { FlutterMethodChannel(name: "multiple-flutters", binaryMessenger: self.engine!.binaryMessenger) }` so it's just non-null

Reply:
Yea, it's a pain but since the initialization has a dependency on `self.engine` I think this is better to have explicit initialization instead of lazy initialization.